admin/src/components/Avatar/styles.ts

```diff
@@ -0,0 +1,15 @@
+// @ts-nocheck
+import {Icon} from "@strapi/design-system/Icon";
+import styled from "styled-components";
+import { Box } from "@strapi/design-system/Box";
+
+
+export const AdminAvatarWrapper = styled(Box)`
+position: relative;
+`;
+
+export const AdminShield = styled(Box)`
```

Comment:
Minot styling improvement

```suggestion
import styled from "styled-components";
import { Box } from "@strapi/design-system/Box";

export const AdminAvatarWrapper = styled(Box)`
    position: relative;
`;

export const AdminShield = styled(Box)`
```